lib/Serialization/ASTReaderStmt.cpp

```diff
@@ -960,23 +960,31 @@ void ASTStmtReader::VisitCountBoundsExpr(CountBoundsExpr *E) {
   VisitExpr(E);
   E->setKind((BoundsExpr::Kind)Record[Idx++]);
   E->setCountExpr(Reader.ReadSubExpr());
-  E->setStartLoc(ReadSourceLocation(Record, Idx));
-  E->setRParenLoc(ReadSourceLocation(Record, Idx));
+  E->StartLoc = ReadSourceLocation(Record, Idx);
```

Comment:
What was the reason to make this change?  Were you thinking that making it private avoided other code from making unsupported changes (at the risk of letting this one piece of code in on the private members)?